src/models/task/postgres.js

```diff
@@ -59,10 +60,32 @@ const TaskRepo = (postgres) => {
         }
     };
 
+    const setTaskCompleteSQL = `
+        UPDATE tasks SET completed=true, timeCompleted=CURRENT_TIMESTAMP
+        WHERE id=$1
+        RETURNING *;
+    `
+    //my work
+    const setTaskCompleted = async (taskID) => {
+        const values = [taskID];
+        try {
+            const client = await postgres.connect(); //client is an object that has a connection to DB 
+            const res = await client.query(setTaskCompleteSQL, values);
+            client.release();
+            return [res.rows[0], null]; //requires null because no error in this case
```

Comment:
So when there was no error, instead of returning `null`, I decided to return an empty string like `""`. @alexlongerbeam, what do you think? For consistency, we should use a standard across all of our endpoints.

Reply:
Yeah lets stick with the standard Jason set so that he doesn't have to go back and re-do his work. I think both approaches are fine as long as we are consistent and document it in the Readme